linker/shared/src/test/scala/org/scalajs/linker/backend/javascript/PrintersTest.scala

```diff
@@ -159,6 +159,13 @@ class PrintersTest {
     )
   }
 
+  @Test def delayedIdentPrintVShow(): Unit = {
```

Comment:
What does the `V` stand for? disjunction?

Reply:
"Versus". I spelled it out completely now.